### PR TITLE
Prep for release 0.7.0

### PR DESCRIPTION
**Pull request by nllong**

### CODE REVIEW

**Comment by DavidGoldwasser** [comment]
@nllong hold on this for a bit. It ran through CI with 3.6.0. I'm seeing local issues on my computer with 3.7.0. I'll re-run the CI which should not use 3.7.0.rc1.

**Comment by DavidGoldwasser** [comment]
@nllong Just re-ran this through the CI with rc2 (which used labs) should be same as rc3. We are going to do official 3.7.0 release today. I can wait until we test with that if you would like to merge this.

I'm also going to file an issue related to running spec tests on different platforms. The CI is passing on Linux, and does locally for @kbenne  when he runs it on his machine. But for me locally on Mac (M1) and on Windows one test fails trying to run bundler within the tests framework.

**Review by kflemin — APPROVED** [review]

**Pull request merged**
